db/migrate/20160127035213_create_seasons.rb

```diff
@@ -0,0 +1,7 @@
+class CreateSeasons < ActiveRecord::Migration
+  def change
+    create_table :seasons do |t|
+      t.date :start_date, null: false
```

Comment:
I would go with `datetime` basically always. I can't think of a reason to use only `date`. Off the top of my head, reasons for using datetime: more specific data is better, also timezones make a difference here.